packages/mock/src/msw/index.ts

```diff
@@ -95,19 +95,23 @@ const generateDefinition = (
     : '';
 
   const delay = getDelay(override, !isFunction(mock) ? mock : undefined);
+  const handlerHasOverride = isReturnHttpResponse && !isTextPlain;
```

Comment:
```suggestion
  const isHandlerOverridden= isReturnHttpResponse && !isTextPlain;
```

 I prefer `is` to `has`